companies/1und1.json

```diff
@@ -1,5 +1,5 @@
 {
-    "slug": "1und1",
+    "slug": "1und2",
```

Comment:
🚫 **[recordlint]** <[match-filename-slug](https://github.com/datenanfragen/data/#data-formats)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Filename `1und1.json` does not match slug `1und2`.

Either rename the file or change the slug.
```suggestion
    "slug": "1und1",
```